src/internal/styles/direction/mixins.scss

```diff
@@ -0,0 +1,20 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+/* stylelint-disable @cloudscape-design/no-implicit-descendant, plugin/no-unsupported-browser-features, selector-combinator-disallowed-list */
+@mixin with-direction($selector, $direction: rtl) {
+  @supports selector(:dir(#{$direction})) {
+    #{$selector}:dir(#{$direction}) {
+      @content;
+    }
+  }
+
+  @supports not selector(:dir(rtl)) {
```

Comment:
Why is this dynamic `:dir(#{$direction})` in the expression before and hard coded here?